src/views/registration-js/attendees/fields/birth.php

```diff
@@ -0,0 +1,96 @@
+ */
+$required    = isset( $field->required ) && 'on' === $field->required ? true : false;
+$option_id   = "tribe-tickets-meta_{$field->slug}_{$ticket->ID}{{data.attendee_id}}";
+$birth_field = $field;
+$field       = (array) $field;
+$field_name  = 'tribe-tickets-meta[' . $ticket->ID . '][{{data.attendee_id}}][' . esc_attr( $field['slug'] ) . ']';
+$disabled    = false;
+$classes     = [ 'tribe-common-b1', 'tribe-field', 'tribe-tickets__item__attendee__field__birth' ];
+
+if ( $required ) {
+	$classes[] = 'tribe-tickets-meta-required';
+}
+?>
+<div class="tribe-horizontal-date-picker-container">
+	<div <?php tribe_classes( $classes ); ?> >
+		<label
+				class="tribe-common-b2--min-medium tribe-tickets-meta-label"
+				for="<?php echo esc_attr( $birth_field->month_id ); ?>"
+		><?php echo wp_kses_post( $field['label'] ); ?><?php tribe_required_label( $required ); ?></label>
+
+		<div class="tribe_day_month_year_datepicker tribe__tickets__item__attendee__field__birth__month">
+			<select
+					<?php tribe_disabled( $disabled ); ?>
+					<?php tribe_required( $required ); ?>
+					class="tribe-horizontal-date-picker-month"
+					onchange="tribe_events_horizontal_date_picker_update_value(this)"
+			>
+				<option value="" disabled selected><?php esc_html_e( "Month", 'tribe-event-plus' ); ?></option>
+				<?php
+				foreach ( $birth_field->get_months() as $month_number => $month_name ) {
+					$month_number = esc_attr( $month_number );
+					$month_name   = esc_attr( $month_name );
+
+					echo "<option value='$month_number'>$month_name</option>";
+				}
+				?>
+			</select>
+		</div>
+		<div class="tribe_day_month_year_datepicker tribe__tickets__item__attendee__field__birth__day">
+			<select
+					<?php tribe_disabled( $disabled ); ?>
+					<?php tribe_required( $required ); ?>
+					class="tribe-horizontal-date-picker-day"
+					onchange="tribe_events_horizontal_date_picker_update_value(this)"
+			>
+				<option value="" disabled selected><?php esc_html_e( "Day", 'tribe-event-plus' ); ?></option>
+				<?php
+				foreach ( $birth_field->get_days() as $day ) {
+					$day = esc_attr( $day );
+
+					echo "<option>$day</option>";
+				}
+				?>
+			</select>
+		</div>
+		<div class="tribe_day_month_year_datepicker tribe__tickets__item__attendee__field__birth__year">
+			<select
+					<?php tribe_disabled( $disabled ); ?>
+					<?php tribe_required( $required ); ?>
+					class="tribe-horizontal-date-picker-year"
+					onchange="tribe_events_horizontal_date_picker_update_value(this)"
+			>
+				<option value="" disabled selected><?php esc_html_e( "Year", 'tribe-event-plus' ); ?></option>
+				<?php
+				foreach ( $birth_field->get_years() as $year ) {
+					$year = esc_attr( $year );
+					echo "<option>$year</option>";
+				}
+				?>
+			</select>
+		</div>
+	</div>
+	<div>
+		<input
+				type="<?php echo isset( $_GET[ 'qa' ] ) ? 'text' : 'hidden' ?>"
+				class="tribe-common-form-control-birth__input ticket-meta tribe-horizontal-date-picker-value"
```

Comment:
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :warning: Processing form data without nonce verification.
* :no_entry_sign: Array keys must NOT be surrounded by spaces if they only contain a string or an integer.

<sub>hash: 49059d06a3396a293e331c36e5a21217</sub>